monai/data/nifti_reader.py

```diff
@@ -135,3 +135,38 @@ def __getitem__(self, index):
                 continue
             compatible_meta[meta_key] = meta_datum
         return img, target, compatible_meta
+
+
+@export("monai.data")
+class Datasetd(Dataset):
+    """
+    General Dataset to handle dictionary format data, it can operate transforms for specific fields.
+    Input data should be a list of dictionaries, for example:
+    [{                            {                            {
+         'img': 'image1.nii.gz',      'img': 'image2.nii.gz',      'img': 'image3.nii.gz',
+         'seg': 'label1.nii.gz',      'seg': 'label2.nii.gz',      'seg': 'label3.nii.gz',
+         'extra': 123,                'extra': 456,                'extra': 789,
+         'shape': 'CHWD'              'shape': 'CHWD'              'shape': 'CHWD'
```

Comment:
the "shape' in the doc can be misleading, could you remove?

Reply:
Updated in the latest patch.
Thanks.